### PR TITLE
Readme updates

### LINKED ISSUE #148: Documentation for setup

### PR DESCRIPTION
**Pull request by jarrydlee**
Fixes https://github.com/PagerNation/PagerNation/issues/148

### CODE REVIEW

**Comment by jarrydlee** [comment]
Added in some steps to create the initial admin user and initial group

File: README.md [hunk 1]
```diff
@@ -4,12 +4,129 @@
...
+
+POST - localhost:3000/api/v1/auth/signup
+
+    `{
+        "email": "test1@test.test",
+        "name": "test",
+        "password": "anything"
+    }`
+
+Get the id andtoken returned with the new user object, we will need it in future steps.
```
Comment (kalebdavis) [inline README.md]: space between 'andtoken'

File: README.md [hunk 2]
```diff
@@ -4,12 +4,129 @@
...
+This user will need to be made the initial system admin in order to have full permissions. To do this, you will need to access to DB and change the `isSystemAdmin` flag on this user to `true`.
+
+**NOTE: For ALL future steps, an auth header must be added to the requests as follows**
+
+    Authorization: Bearer <auth token from step 1>
+
+**2. Create a group**
+
+POST - localhost:3000/api/v1/group
```
Comment (kalebdavis) [inline README.md]: Same note as above: This might be more readable if it's also in the code block

File: README.md [hunk 3]
```diff
@@ -4,12 +4,129 @@
...
+POST - localhost:3000/api/v1/auth/signup
+
+    `{
+        "email": "test1@test.test",
+        "name": "test",
+        "password": "anything"
+    }`
+
+Get the id andtoken returned with the new user object, we will need it in future steps.
+This user will need to be made the initial system admin in order to have full permissions. To do this, you will need to access to DB and change the `isSystemAdmin` flag on this user to `true`.
```
Comment (kalebdavis) [inline README.md]: `isSysAdmin` not `isSystemAdmin`

**Review by kalebingerick — CHANGES_REQUESTED** [review]
The set up portion is great, if that's all this PR is then once the few comments I have are addressed it should be good to go.

I would like to see more information in the Running, Tests, and Seeding sections, but those could be updated in another PR

**Pull request closed without merging**